UnityProject/Assets/Scripts/Player/PlayerMove.cs

```diff
@@ -21,16 +21,23 @@ public class PlayerMove : NetworkBehaviour, IRightClickable, IServerSpawn
 	[SyncVar] public bool allowInput = true;
 
 	//netid of the game object we are buckled to, NetId.Invalid if not buckled
-	[SyncVar(hook = nameof(OnBuckledChangedHook))]
-	public uint buckledObject = NetId.Invalid;
+	[SyncVar(hook = nameof(SyncBuckledObjectNetId))]
+	private uint buckledObjectNetId = NetId.Invalid;
```

Comment:
I don't know if NetId.Empty would make more sense than NetId.Invalid to represent a "not buckled" state?